superset-frontend/src/SqlLab/components/ExploreResultsButton.jsx

```diff
@@ -139,8 +139,8 @@ class ExploreResultsButton extends React.PureComponent {
           datasource: `${data.table_id}__table`,
           metrics: [],
           groupby: [],
+          time_range: 'No filter',
```

Comment:
a bit weird that this is just plain text as opposed to `null` or something. I don't think fixing that is required right now, but i think moving "No filter" to a constant and using that constant it here would be reasonable. It looks like "No filter" is used in a few other places too (tests, other components) so maybe you could replace those your new constant too?

Reply:
Agree this is not a good practice. There seems to be many other hard-coded time periods, as well. I am not comfortable with refactoring while changing the default or refactoring just the `'No filter'`. Maybe that refactoring can happen in a later PR?